CSS/style.css

```diff
@@ -0,0 +1,574 @@
+  margin-top: 30px;
+}
+
+.hor {
+  height: 1px;
+  width: 45px;
+  margin-top: 2px;
+  background-color: #ec5242;
+  border: none;
+  margin-bottom: 20px;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  top: 50px;
+}
+
+.flex-box {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  background-color: rgba(255, 255, 2555, 0.1);
+  margin: 10px;
+}
+
+.lecture {
+  font-family: 'lato', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 24px;
+  color: #ec5242;
+}
+
+.details {
+  font-family: 'lato', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 24px;
+  color: white;
+}
+
+.btn {
+  height: 60px;
+  width: 327px;
+  background: #ec5242;
+  font-family: 'lato', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: aliceblue;
+  margin-top: 20px;
+  border: none;
+  margin-bottom: 20px;
+}
+
+.people {
+  width: 100%;
+}
+
+.h {
+  text-align: center;
+  font-family: 'lato', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 24px;
+  line-height: 24px;
+  color: #272a31;
+  margin-top: 50px;
+}
+
+.speakerhor {
+  height: 1px;
+  width: 45px;
+  background-color: #ec5242;
+  margin: 8px auto 50px auto;
+  border: none;
+}
+
+.speaker-container {
+  display: flex;
+  flex-direction: column;
+  padding-top: 25px;
+  padding-bottom: 10px;
+  margin: 20px;
+}
+
+.listItem {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  width: 100%;
+  float: left;
+}
+
+.speak {
+  gap: 2px;
+  width: 100%;
+}
+
+.name {
+  font-family: 'lato', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 10px;
+  color: #ec5242;
+}
+
+.title {
+  font-family: 'lato', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.mydiv {
+  border: 2px solid orange;
+}
+
+.partners {
+  width: 100%;
+  background-color: rgb(65, 64, 64);
+  display: flex;
+  flex-direction: column;
+}
+
+.pat {
+  height: 40px;
+}
+
+.partners1 {
+  font-family: 'lato', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: white;
+  text-align: center;
+  margin-bottom: auto;
+}
+
+.hor2 {
+  height: 1px;
+  width: 45px;
+  margin-top: auto;
+  background-color: #ec5242;
+}
+
+.line {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2px;
+}
+
+.list {
+  list-style: none;
+  display: flex;
+  align-items: center !important;
+  gap: 20px;
+  font-family: 'lato', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 40px;
+  color: antiquewhite;
+  text-align: center;
+  margin-top: 70px;
+}
+
+.list li {
+  align-items: center;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: 'lato', sans-serif;
```

Comment:
- [ ] Hi you are required to stick to the design as much as possible so kindly give your footer a `flex-direction: row;` with some space between the logo and the text as it is on the provided design

**your design**
<img width="1436" alt="image" src="https://user-images.githubusercontent.com/37901469/192102680-ca5cb195-3288-4396-84bd-84446f3f924a.png">

**DTF(Design to follow)**
<img width="983" alt="image" src="https://user-images.githubusercontent.com/37901469/192102733-b9901d96-2871-4d28-b4b9-eed899ecb107.png">